Documentation/Tools/HolographicRemoting.md

```diff
@@ -115,21 +111,29 @@ You can also temporarily remove the adapter to workaround your issue via the fol
 1. (Optional) Navigate to MRTK / Providers / WindowsMixedReality / Shared / DotNetAdapter in your file explorer (not Unity's Assets view) and delete the `.bin` and `.obj` folders. This removes the local cache of NuGet restored packages for DotNetWinRT.
 1. If you run the MRTK Configurator again, make sure you don't re-enable MSBuild for Unity.
 
+### XR SDK setup instructions
+
+1. Open **Window > Package Manager**
+    - Verify that latest version of the **Windows Mixed Reality** package is installed.
+1. Go to the Standalone settings and ensure Windows Mixed Reality is checked under Plug-in Providers. Also ensure that Initialize XR on Startup is checked.
+1. (**_Optional_**) Click on the Windows Mixed Reality tab under XR Plug-in Management and create a custom settings profile to change the defaults. If the list of settings are already there, no profile needs to be created.
+1. Select **Mixed Reality Toolkit** > **Utilities** > **Windows Mixed Reality** > **Check Configuration** to make sure the correct scripting define is set.
```

Comment:
We might want to make it clear that DotNetWinRT isn't required for HL2 features on XR SDK. I'm not sure this step is needed